MilitaryManager.Attachments.API/Controllers/DecreesController.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using MilitaryManager.Core.DTO.Attachments;
+using MilitaryManager.Core.Entities.DecreeEntity;
+using MilitaryManager.Core.Interfaces.Services;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace MilitaryManager.Attachments.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class DecreesController : ControllerBase
+    {
+        private readonly IDecreeService _decreeService;
+        private readonly string _webRootPath;
+
+        public DecreesController(IDecreeService decreeService,
+                                IWebHostEnvironment hostingEnvironment)
+        {
+            _decreeService = decreeService;
+            _webRootPath = hostingEnvironment.WebRootPath;
+        }
+
+        [HttpGet]
+        [Route("collection")]
+        public async Task<ActionResult<IEnumerable<DecreeDTO>>> Get()
+        {
+            var decrees = await _decreeService.GetDecreesAsync();
+            return Ok(decrees);
+        }
+
+        [HttpGet("collection/byName/{name}")]
```

Comment:
Check the HTTP service in client side in order to make the routes accordingly

Reply:
Create new method in base.service.ts?